src/quo/components/buttons/slide_button/view.cljs

```diff
@@ -33,7 +32,8 @@
                  theme
                  type
                  blur?]}]
-      (let [x-pos             (reanimated/use-shared-value 0)
+      (let [gestures-disabled? (reagent/atom disabled?)
```

Comment:
I think reagent atoms should live outside the function in [Form 2 components](https://github.com/reagent-project/reagent/blob/master/doc/CreatingReagentComponents.md#form-2--a-function-returning-a-function). Maybe instead of moving the ratom inside the fn, the atom could be updated from a `use-effect` hook whenever the prop changes?

Reply:
You are right @clauxx, fixed!